Add a Code of Conduct to the website

Hamatti opened the pull request:
Hello community!

I think it's about to do something that has been long overdue. I want to introduce a Code of Conduct into our events.

Nothing major has come to my knowledge about incidents in our events but we shouldn't wait for bad to happen. Building a welcoming community where everyone gets treated fairly and with respect has always been a personal guideline of mine.

But these things also need to be said out loud and made visible to everyone. That's why I'm proposing this Code of Conduct that is copied from [Berlin Code of Conduct](https://berlincodeofconduct.org/) that is a very widely used CoC in the industry.

I do want to start discussion about this and offer you, the members of this community, to chime in with your opinions because after all, it's your community. Especially I'm interested in if there's something we should add or clarify in the contents of the CoC to make it even better for us.

Hamatti committed:
Add CoC

Haprog commented on code-of-conduct.md:
```diff
@@ -0,0 +1,62 @@
+---
+# You don't need to edit this file, it's empty on purpose.
```
Should this comment be removed or updated? It doesn't seem to be true. This file is not empty and has relevant content. Or is the comment referring to the `layout: coc`? In any case this is a bit confusing to me.

Hamatti replied:
Ha, good point! Classic copy-pasting from old stuff.

Haprog replied:
This is still there in this file. You removed it only from `index.md`.

Haprog commented on code-of-conduct.md:
```diff
@@ -0,0 +1,62 @@
+
+## If You Witness or Are Subject to Unacceptable Behavior
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community organizer as soon as possible. You can find a list of organizers to contact for each of the supporters of this code of conduct at the bottom of this page. Additionally, community organizers are available to help community members engage with local law enforcement or to otherwise help thos ...
```
> You can find a list of organizers to contact for each of the supporters of this code of conduct at the bottom of this page.

I don't see a list of organizers in this file. Will it be part of the live page or should this section or file be updated related to contacts?

Hamatti replied:
Very good find, this is what happens with copy-pasting, I totally forgot to add our information there.

Haprog commented on index.md:
```diff
@@ -9,4 +9,6 @@ Turku &hearts; Frontend is the #1 community for frontend developers and designer
-Our meetups would not be possible without our [amazing speakers](/speakers)
+Our meetups would not be possible without our [amazing speakers](/speakers).
+
+Please note that everyone on our meetups are expected act in accordance with the [Code of Conduct](/code-of-conduct) so we can build a community that is open and welcoming to everyone.
```
Some typos here. I think it should be "everyone in our meetups is expected to act" instead of "everyone on our meetups are expected act".

Hamatti replied:
Thanks!

Haprog commented on css/main.scss:
```diff
@@ -1,50 +1,54 @@
 ---
-
```
This file seems to have mostly code style changes unrelated to the "Code of Conduct" PR topic. I guess it's fine but I would have made these changes outside of this PR for clarity.

Hamatti replied:
Prettier being prettier. Basically the only change on that file is at the bottom where I added some styling and text-align for code of conduct page.

I'm gonna leave these changes in for now as there's very little relevant changes for review in the file.

Hamatti committed:
Add organizer contact info and fix small things

- Add Juhis, Teemu and Matias as organizers
- Fix mistakes in spelling
- Remove Jekyll default "don't edit this file" comments

Haprog commented on index.md:
```diff
@@ -1,12 +1,11 @@
-Our meetups would not be possible without our [amazing speakers](/speakers)
+Our meetups would not be possible without our [amazing speakers](/speakers).
+
+Please note that everyone on our meetups is expected act in accordance with the [Code of Conduct](/code-of-conduct) so we can build a community that is open and welcoming to everyone.
```
> Please note that everyone on our meetups is expected act in accordance with

Still typos here. I'd use "in" instead of "on" though I'm not 100% sure which is more correct here.

But in any case there's at least a "to" missing from "is expected _to_ act"

Haprog commented:
> # You don't need to edit this file, it's empty on purpose.

This is still there in `code-of-conduct.md`. Wanted to mention it in a new comment since it shows up as "resolved" and collapsed above even though I added a new comment on it.

Hamatti commented:
> > # You don't need to edit this file, it's empty on purpose.
> 
> This is still there in `code-of-conduct.md`. Wanted to mention it in a new comment since it shows up as "resolved" and collapsed above even though I added a new comment on it.

Oh man, did I remove it from a wrong file :D

Hamatti committed:
Remove Jekyll 'don't edit this' boilerplate

Hamatti committed:
Fix typo

Haprog commented:
@Hamatti Thanks for the updates! Looks pretty good now 👍
